src/vaadin-tooltip.html

```diff
@@ -60,15 +60,32 @@
         static get properties() {
           return {
             /**
-             * The text content of the tooltip
+             * The id of the target element. Must be a sibling.
              */
             for: {
               type: String,
               observer: '_attachToTarget'
+            },
+            /**
+             * Tooltip position. Possible values: top, right, left and bottom.
+             */
+            position: {
+              type: String,
+              observer: '_setPosition',
+              value: 'top'
             }
           };
         }
 
+        connectedCallback() {
```

Comment:
It's just an empty method, maybe it makes sense to implement it when you need it?